docs/tutorials/gstreamer.mdx

```diff
@@ -1 +1,100 @@
+---
+sidebar_position: 4
+---
 
+# Build GStreamer pipeline with opencv-python 
+
+This tutorial demonstrates how to use OpenCV with GStreamer to capture video from cameras. It supports different camera configurations, pixel formats, and frame size.
+
+## Tutorial
+
+### Prerequisites
+1. opencv-python with gstreamer backend
+3. **windows**:  
+    Install the C++ version of sensing-dev. make sure GST_PLUGIN_PATH is set.
+3. **linux**:  
```

Comment:
can you mention `--install-gst-tools` here too?

Reply:
done